crates/bevy_sprite/src/rect.rs

```diff
@@ -12,6 +12,20 @@ pub struct Rect {
     pub max: Vec2,
 }
 
+/// Struct defining a [`Sprite`](crate::Sprite) border with padding values
+#[repr(C)]
```

Comment:
Why do we need this?

Reply:
Copied and pasted from `Rect`, I'll remove it